packages/expo-web-browser/src/WebBrowser.ts

```diff
@@ -249,7 +249,7 @@ export function dismissBrowser(): void {
  */
 export async function openAuthSessionAsync(
   url: string,
-  redirectUrl: string,
+  redirectUrl?: string,
```

Comment:
Maybe I should make this also nullable?
```suggestion
  redirectUrl?: string | null,
```

Reply:
Yes, I think we should allow both `undefined` and `null` on everything that is optional.